xbmc/video/Teletext.cpp

```diff
@@ -1285,7 +1285,7 @@ void CTeletextDecoder::RenderPage()
         else
         {
           SetPosX(33+i);
-          m_RenderInfo.PageChar[32+i] = m_RenderInfo.PageChar[32+i];
+          m_RenderInfo.PageChar[32+i];
```

Comment:
you need to remove this line completely as it has no effect

Reply:
I removed and only leave 
![image](https://user-images.githubusercontent.com/45079270/98145167-2092ed80-1ea1-11eb-8179-512c5dd947b6.png)

Reply:
yes, but this statement `m_RenderInfo.PageChar[32+i];` does nothing and will generate a new warning on the build system. the original line also did nothing, just the self-assignment, but didn't change the value
so it's fine if you remove it entirely and just go with
```c
else
{
   SetPosX(33+i);
}
```

Reply:
Alright, thank you. I will push again.

Reply:
Check now